packages/fxa-payments-server/src/routes/Product/SubscriptionCreate/index.tsx

```diff
@@ -89,6 +89,8 @@ export const SubscriptionCreate = ({
   coupon,
   setCoupon,
 }: SubscriptionCreateProps) => {
+  selectedPlan.checkoutType = 'with-account';
```

Comment:
added the type in fxa-shared to avoid a circular reference between fxa-shared and fxa-payments-server since i added checkoutType as a property on Plan (explained below - it was basically already there so that metrics data could be piggy-backed on the plan object)